### PR TITLE
Allow app developers to specify the minimum int size

### PR DESCRIPTION
**Pull request by BernhardPosselt**
Example for requiring a 64bit or higher PHP in your info.xml:

``` xml
<?xml version="1.0"?>
<info>
    <dependencies>
        <php min-int-size="64"/>
    </dependencies>
</info>
```

### CODE REVIEW

File: tests/lib/app/dependencyanalyzer.php [hunk 1]
```diff
@@ -278,13 +285,14 @@ function providesDatabases() {
...
-			array(array('PHP with a version lower than 5.4.2 is required.'), null, '5.4.2'),
-			array(array(), '5.4', '5.4'),
+			array(array(), null, null, null),
+			array(array(), '5.4', null, null),
+			array(array(), null, '5.5', null),
+			array(array(), '5.4', '5.5', null),
+			array(array('PHP 5.4.4 or higher is required.'), '5.4.4', null, null),
+			array(array('PHP with a version lower than 5.4.2 is required.'), null, '5.4.2', null),
+			array(array('64bit or higher PHP required.'), null, null, 64, null),
+			array(array(), '5.4', '5.4', null),
```
Comment (nickvergessen) [inline tests/lib/app/dependencyanalyzer.php]: Also add tests with intSize !== null?

Comment (BernhardPosselt) [inline tests/lib/app/dependencyanalyzer.php]: one line further up

Comment (BernhardPosselt) [inline tests/lib/app/dependencyanalyzer.php]: one null too much there however :)

**Comment by BernhardPosselt** [comment]
Tested locally with 65bit and 64bit in News app and works :)

Fixed an off by one error

**Comment by PVince81** [comment]
Yeah, why not :+1:
